scripts/secbugsreport.pl

```diff
@@ -45,32 +45,59 @@
     products     => $products,
     sec_keywords => $sec_keywords
 );
+
 my $vars = {
     urlbase         => Bugzilla->localconfig->{urlbase},
     report_week     => $report_week,
     products        => $products,
     sec_keywords    => $sec_keywords,
     results         => $report->results,
+    deltas          => $report->deltas,
     build_bugs_link => \&build_bugs_link,
 };
 
+my $x = $report->deltas;
+
+
 $template->process( 'reports/email/security-risk.html.tmpl', $vars, \$html )
     or ThrowTemplateError( $template->error() );
 
+
+# TODO: hacky, remove me!
```

Comment:
currently have a valid png file in /data but seem to not be able to load the file :/
it is properly attached on the email but is only 5 bytes. I suspect I have to loop through something